api/api.go

```diff
@@ -182,6 +185,7 @@ func ReadJSONBody(ctx context.Context, body io.ReadCloser, v interface{}, w http
 
 	// Set headers
 	w.Header().Set("Content-Type", "application/json; charset=utf-8")
+	w.Header().Set("Cache-Control", "public, max-age=1800")
```

Comment:
Surely the max-age value should use the configured max age: `cfg.NavigationCacheMaxAge.Seconds()`

Another issue with this is that the ReadJSONBody function is used by all Read endpoints for the API so is not specific to navigation cache max age. Therefore it might make sense to not add this into this function and do it only against the Navigation endpoint.